js/localstorage.js

```diff
@@ -0,0 +1,23 @@
+function localData() {
+  const localData = {
+    name: document.getElementById('savename').value,
+    email: document.getElementById('saveemail').value,
+    message: document.querySelector('textarea').value,
+  };
+  localStorage.setItem('localData', JSON.stringify(localData));
+}
+
+const Inputs = document.querySelectorAll('#form input, #form textarea');
+Inputs.forEach((input) => {
+  input.addEventListener('input', localData);
+});
```

Comment:
👋🏻  Thank you for your efforts in addressing the previous review comments. However, a new error has arisen in your code. Previously, you had an email input field with the ID "save email," which you updated to "email" after the review feedback. Now, when attempting to add data, an error occurs: "Cannot read properties of null (reading 'value') at HTMLInputElement.localData (local storage.js:4:48)."


[256900114-b402b9b4-e871-42f3-a3a0-af81271fa5ac.webm](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/66476329/d60ff576-41fa-4767-bb9a-b9109ca2071c)

Please resolve this issue 🙏, as it is preventing me from checking the updates made by the previous reviewer. By resolving this error, you'll be able to proceed with the further review of your changes. Keep up the great work, and happy coding! 💻🌟🚀

Reply:
Thank you very much for your suggestion, I have made the requested changes, I thank you for your time.👍